catalogue/webapp/components/WorkDetails/WorkDetails.js

```diff
@@ -29,6 +29,27 @@ import { clientSideSearchParams } from '@weco/common/services/catalogue/search-p
 import Download from '../Download/Download';
 import PaletteSimilarityBox from '../PaletteSimilarityBox/PaletteSimilarityBox';
 
+type TimeProps = {|
+  hours: string,
+  minutes: string,
+  seconds: string,
+|};
+
+function doubleDigits(n) {
+  return n > 9 ? `${n}` : `0${n}`;
```

Comment:
Could use `padStart` and add it to polyfil for ie?

Reply:
I wasn't even aware `padStart` was a thing. Nice.